manifests/s/Stretchly/Stretchly/1.11.0/Stretchly.Stretchly.installer.yaml

```
@@ -0,0 +1,25 @@
+PackageIdentifier: Stretchly.Stretchly
+PackageVersion: 1.11.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
```

```suggestion
- Architecture: x64
```
This application only runs on 64-bit version of Windows.